retriever/lib/retrieval.js

```diff
@@ -13,7 +13,7 @@
  *   - The response from the fetch request, the cache miss and the content length.
  */
 export async function retrieveFile(baseUrl, pieceCid, cacheTtl = 86400) {
-  const url = `https://${baseUrl}/piece/${pieceCid}`
+  const url = `${baseUrl}/piece/${pieceCid}`
```

Comment:
```suggestion
  const url = `${baseUrl}/piece/${rootCid}`
```

Maybe let's rename `pieceCid` to `rootCid` here to keep it consistent with rest of the codebase.